index.js

```diff
@@ -98,6 +98,39 @@ function QuickBooks(consumerKey, consumerSecret, token, tokenSecret, realmId, us
   if (!tokenSecret && this.oauthversion !== '2.0') throw new Error('tokenSecret not defined');
 }
 
+/**
+ *
+ * Use the refresh token to obtain a new access token.
+ *
+ *
+ */
+
+QuickBooks.prototype.refreshAccessToken = function(callback) {
+    var auth = (new Buffer(this.consumerKey + ':' + this.consumerSecret).toString('base64'));
+
+    var postBody = {
+        url: 'https://oauth.platform.intuit.com/oauth2/v1/tokens/bearer',
+        headers: {
+            Accept: 'application/json',
+            'Content-Type': 'application/x-www-form-urlencoded',
+            Authorization: 'Basic ' + auth,
+        },
+        form: {
+            grant_type: 'refresh_token',
+            refresh_token: this.refreshToken
+        }
+    };
+
+    request.post(postBody, function (e, r, data) {
+        var refreshResponse = JSON.parse(r.body);
+        console.log(refreshResponse);
+        this.refreshToken = refreshResponse.refresh_token;
+        this.token = refreshResponse.access_token;
+        callback(refreshResponse, e);
```

Comment:
Before this is out in the wild too long, is there a reason for inverting the [error-first callback standard](https://nodejs.org/api/errors.html#errors_error_first_callbacks)?

Reply:
no good reason? We could invert it. Does the rest of the library conform?